k8s/eks-env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if ! command -v jq &> /dev/null
+then
+    echo "jq could not be found"
+    exit
+fi
+
+if ! command -v yq &> /dev/null
+then
+    echo "yq could not be found"
+    exit
+fi
+
+id=$(aws sts get-caller-identity)
+userarn=$(echo ${id} | jq '.Arn')
+accountid=$(echo ${id} | jq --raw-output '.Account')
+username=$(kubectl get -o json -n kube-system configmap aws-auth \
+	| jq --raw-output '.data.mapUsers' \
+	| yq eval ".[] | select(.userarn == $userarn) | .username" -)
```

Comment:
Nit: If you assume the AWS and Kubernetes users are the same (a reasonable requirement, I think), you can simplify to just
```sh
account=$(aws sts get-caller-identity --output text --query Account)
arn=$(aws sts get-caller-identity --output text --query Arn)
user=${arn#*/}
```

Reply:
Great!